projects/netgrif-components/schematics/view/create-view-prompt/views/public-task-view/files/common/__className@dasherize__.component.ts.template

```diff
@@ -0,0 +1,135 @@
+import {
+    AbstractTaskViewComponent,
+    TaskEventNotification,
+    TaskViewService,
+    CaseResourceService,
+    PublicCaseResourceService,
+    PublicTaskResourceService,
+    SearchService,
+    PublicProcessService,
+    ProcessService,
+    TaskResourceService,
+    SnackBarService,
+    UserService,
+    SessionService,
+    PetriNetResourceService,
+    PublicPetriNetResourceService,
+    LoggerService,
+    ResourceProvider,
+    ConfigurationService,
+    FieldConverterService,
+    AuthenticationService,
+    PublicUrlResolverService,
+    publicBaseFilterFactory,
+    publicFactoryResolver,
+    AllowedNetsService,
+    PublicTaskLoadingService,
+    AllowedNetsServiceFactory,
+    NAE_VIEW_ID_SEGMENT,
+    ViewIdService,
+    NAE_BASE_FILTER,
+    ChangedFieldsService
+} from '@netgrif/components-core';
+import {HeaderComponent} from '@netgrif/components';
+import {ActivatedRoute, Router} from '@angular/router';
+import {TranslateService} from '@ngx-translate/core';
+import {combineLatest} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+const localAllowedNetsServiceFactory = (factory: AllowedNetsServiceFactory, route: ActivatedRoute) => {
+    const array = [];
+    if (route.snapshot.paramMap.get('petriNetId') !== null) {
+        array.push(route.snapshot.paramMap.get('petriNetId'));
+    }
+    return factory.createFromArray(array);
+};
+
+const processServiceFactory = (userService: UserService, sessionService: SessionService, authService: AuthenticationService,
+                               router: Router, publicResolverService: PublicUrlResolverService, petriNetResource: PetriNetResourceService,
+                               publicPetriNetResource: PublicPetriNetResourceService, loggerService: LoggerService, redirectService: RedirectService) => {
+    return publicFactoryResolver(userService, sessionService, authService, router, publicResolverService,
+        new ProcessService(petriNetResource, loggerService),
+        new PublicProcessService(publicPetriNetResource, loggerService), redirectService);
+};
+
+const taskResourceServiceFactory = (userService: UserService, sessionService: SessionService, authService: AuthenticationService,
+                                    router: Router, publicResolverService: PublicUrlResolverService,
+                                    logger: LoggerService, provider: ResourceProvider, config: ConfigurationService,
+                                    fieldConverter: FieldConverterService, redirectService: RedirectService) => {
+    return publicFactoryResolver(userService, sessionService, authService, router, publicResolverService,
+        new TaskResourceService(provider, config, fieldConverter, logger),
+        new PublicTaskResourceService(provider, config, fieldConverter, logger), redirectService);
+};
+
+const caseResourceServiceFactory = (userService: UserService, sessionService: SessionService, authService: AuthenticationService,
```

Comment:
factory re-use same as in single task view

Reply:
should be in another task